### DISCUSSION TITLE
Option to Update Plex Library per file (like Kodi notifier can)

### DISCUSSION THREAD

**Discussion by desepticon** [pymedusa/Medusa#6358] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
Right now, Medusa seems to only be able to update the entire Plex library during post-processing.

**Describe the solution you'd like**
It would be nice if the notifier could update on a per-file basis like the Kodi notifier can. I fairly certain this is possible. I do know "Plex Media Scanner --scan  --directory <path>" is a usable command line option.

**Describe alternatives you've considered**
none

**Additional context**
It can take a long time to update an entire library if it is large. This would be a nice feature to save both time, io, and processing power.

**Comment by flick1999** [comment]
Would love to see this as well. The CLI approach is being deprecated, though. The new way is via URL commands: https://support.plex.tv/articles/201638786-plex-media-server-url-commands/
